src/lib/OTA/OTA.h

```diff
@@ -92,7 +92,7 @@ typedef struct {
         struct {
             uint8_t packetType: 2,
                     telemetryStatus: 1,
-                    uplinkPower: 3, // PowerLevels_e
+                    uplinkPower: 3, // PowerLevels_e - 1 (1-8 is 0-7 in the air)
```

Comment:
So this really isn't PowerLevels_e then is it? The comment should be
`CRSF_power_level - 1 (1-8 is 0-7 in the air)`

Reply:
(embarrassed paw over face meme) you're right! I'll fix that and the test